unit-tests/unit-test-default-main.cpp

```diff
@@ -36,6 +36,12 @@ int main( int argc, char * argv[] )
                  We write the value directly to 'test::context'.
              */
              | Opt( test::context, "context" )["--context"]( "Context in which to run the tests" )
+             /*
+                 --debug
+
+                 Enable logging of "debug" messages (using test::log.d())
+             */
+             | Opt( test::debug )["--debug"]( "Enable debug logging via test::log.d(...)" )
```

Comment:
Is this the assignment?

Reply:
Yes, that assigns. Not sure if it puts `false` if it does not encounter `--debug`, though